packages/file/test/common_tests.dart

```diff
@@ -2225,6 +2225,7 @@ void runCommonTests(
 
           setUp(() {
             f = fs.file(ns('/foo'));
+            f.createSync(recursive: true);
```

Comment:
How is this related? Can you mention it in the commit message?

Reply:
Fixing CI. This is unrelated, this was an existing broken test.